recipes/statmorph/meta.yaml

```diff
@@ -0,0 +1,49 @@
+
+{% set name = "statmorph" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "5e92600867b3ddbd3bd1194ca8689c5e99e214d909588e29860c05be5f14712e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - scikit-image >=0.13
+    - astropy >=2.0
+    - photutils >=0.3
+  run:
```

Comment:
Add an explicit `python` requirement to `run`, please. `conda` uses it sometimes.